### PR TITLE
add autoscale_test

### PR DESCRIPTION
**Pull request by yarntime**
**What this PR does / why we need it**: add autoscale_test.go

**Release note**:

```NONE
```

### CODE REVIEW

File: pkg/kubectl/autoscale_test.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		if test.expectErr && err != nil {
+			continue
+		}
+		if !reflect.DeepEqual(obj.(*autoscaling.HorizontalPodAutoscaler), test.expected) {
+			t.Errorf("\nexpected:\n%#v\nsaw:\n%#v", test.expected, obj.(*autoscaling.HorizontalPodAutoscaler))
+		}
+	}
+}
+
+func newInt32(val int32) *int32 {
```
Comment (brendandburns) [inline pkg/kubectl/autoscale_test.go]: use `util.Int32Ptr` (https://github.com/kubernetes/kubernetes/blob/master/pkg/util/util.go#L109)

**Comment by brendandburns** [comment]
@k8s-bot ok to test.

Thanks for the PR, two minor comments.

**Comment by yarntime** [comment]
@brendandburns , I have optimized the code, thanks.

**Comment by deads2k** [comment]
need to update bazel.  hack/update-bazel.sh.

**Pull request closed without merging**